pkg/apis/serving/v1alpha1/trainedmodel_webhook_test.go

```diff
@@ -96,9 +96,9 @@ func TestValidateCreate(t *testing.T) {
 		"invalid storageURI prefix": {
 			tm: makeTestTrainModel(),
 			update: map[string]string{
-				storageURI: "https://kfserving/sklearn/iris",
+				storageURI: "pvc://kfserving/sklearn/iris",
```

Comment:
Should replace this by an obviously fake schema (i.e. `foo://`)?

Reply:
I wanted to set the prefix to one of the non-implemented protocols in [Provider.go](https://github.com/kubeflow/kfserving/blob/c1b9fa07d1e6bee4ad2fde2d7cb394089c29b732/pkg/agent/storage/provider.go#L25) but this makes sense as well. Will make that change to avoid future problems.